Configuration/UTMAppleConfiguration.swift

```diff
@@ -270,6 +270,8 @@ extension UTMAppleConfiguration {
                 }
                 if #available(macOS 13, *), drive.isExternal {
                     return VZUSBMassStorageDeviceConfiguration(attachment: attachment)
+                } else if #available(macOS 14, *), drive.isNvme, system.boot.operatingSystem == .linux {
```

Comment:
Does it mean that NVMe will only actually work for Linux and for other OSes it will be ignored?

Reply:
Yes, and on other operating systems (currently only macOS), this option will be ignored, as the NVMe interface does not work with macOS VMs anyway. Adding this ensures that a misconfigured disk does not cause errors.

Reply:
Maybe we should hide this option for other OSes to avoid any confusion in the future?

Reply:
Initially I want to do this as well. However, the data structure currently makes it a little inconvenient to get the current OS because the view itself does not have the full config. Currently we have other options like the following does the similar thing.
```swift
Toggle("HiDPI (Retina)", isOn: isHidpi)
                .help("Only available on macOS virtual machines.")
```

Reply:
Sounds great.